README.md

```diff
@@ -169,6 +170,30 @@ window.ai.addEventListener((event: EventType, data: unknown) => {
 })
 ```
 
+**(BETA) Generate 3D Objects**: generate a 3D object using a specified model or the user-preferred model.
```

Comment:
instead of `using a specified model or the user-preferred model` let's say ". Uses [Shape-E](https://github.com/openai/shap-e)"

Reply:
changed!